app/forms/provider_interface/edit_user_permissions_wizard.rb

```diff
@@ -0,0 +1,42 @@
+module ProviderInterface
+  class EditUserPermissionsWizard
+    include ActiveModel::Model
+
+    attr_accessor :permissions
```

Comment:
Not being used for anything at this stage of the commit.

Reply:
We decided to put the wizard and its unit test in a separate commit. This accessor is used further on though!